staging/src/k8s.io/kube-openapi/pkg/aggregator/aggregator.go

```diff
@@ -320,6 +357,8 @@ func equalIntPointer(s1, s2 *int64) bool {
 
 // EqualSchema returns true if models have the same properties and references
 // even if they have different documentation.
+// Note: Dependencies and Enum are not supported. EqualSchema will return
```

Comment:
I don't expect we use `Dependencies`. For `Enum` it is an array of interfaces. I am not sure how to compare them. Anyway, I don't expect to use that either any time soon and also not implementing them is not terrible. It is a False positive that will result in a unnecessary schema duplication. I can spend time figure out how to implement them, but I don't think it worth it.